FiveCalls/FiveCalls/IssuesContainerViewController.swift

```diff
@@ -65,13 +62,56 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
     }
 
     func editLocationViewController(_ vc: EditLocationViewController, didSelectZipCode zip: String) {
+        dismiss(animated: true, completion: { //`updateWith` must be in completion block so that VC is listening for notification
+            self.updateWith(zipCode: zip)
```

Comment:
Thanks. I actually hadn't fully considered the [weak self] issue. I've been making an unexamined assumption that Swift (3) was magically taking care of that for me.